src/MarkDown/Tos.md

```diff
@@ -0,0 +1,63 @@
+# Terms of Use
+
+## Agreement to our Terms
+
+MovieMentor ("Application") is an application developed and maintained by Jeffrey DiVincent, Adam Fernandes, Jacob Franz, Nathanael Gaulke, Lauren Morris, and Garrett Spears (the "Developers," "we," "us," "our"). We operate the website moviementor.app (the "Site," "Website"), a suite of mobile applications (the "Apps"), as well as other related products and services that fall under these terms (the "Terms") (collectively, the "Services").
+
+MovieMentor can be contacted by email at moviementor@divi.sh.
+
+These Terms constitute a legally binding agreement between you, whether personally or on behalf of an entity ("you"), and the Developers, concerning your use and access to the Services. You agree that by accessing the Services, you have read, understood, and agreed to be bound by all of these Terms.
+
+If you do not agree with any or all of these Terms, you are not permitted to use the Services and must immediately continue use.
+
+The Application and its Developers reserve the right to amend these Terms at any time. Updated Terms will be listed on the Site and be indicated by updating the "Last updated" date on these Terms, and you waive the right to receive specific notice of each such change. It is your responsibility to periodically review these Terms to stay informed up updates. Terms updates are effective immediately on publication.
+
+## Our Services
+
+The information provided when using the Services is not intended for distribution to or use by any person or entity in any jurisdiction or country where such use would be contrary to law or regulation or which subject us to any registration requirement within such jurisdiction or country. Accordingly, those persons who choose to access the Services in other locations do so on their own initiative and are soley responsible for compliance with local laws, if and to the extent local laws are applicable.
+
+MovieMentor should not be used by government agencies where secrecy is required. MovieMentor is also not a medical application; while all user data is stored on-device, we are not beholdent to the Health Insurance Portability and Accountability Act (HIPAA).
```

Comment:
beholdent is not a word from what I could see online, beholden might have been the word you were trying to use?

Reply:
probably yeah